client/public/restaurant.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>A Restaurant</title>
+</head>
+<body>
+    <div id="body">
+    <h4 id="name">Restaurant name</h4>
+    <h5>Hours:<span id="hours">hours</span></h5>
+    <h4 id="address">Address</h4>
+    <h3 id="phone">phone number</h3>
+    <h3 id="notes">Notes</h3>
+    <h3 id="menu">Menu</h3>
+    </div>
+
+    <script>
+        let restaurantId = document.location.pathname.split('/').splice(-1);
+
+        fetch('/api/' + id + '.json')
+            .then((response) => response.json())
+            .then(fillRestaurant)
+
+            function fillRestaurant(restaurant) {
+                document.getElementById('name').textContent = restaurant.name;
+                document.getElementById('hours').textContent = restaurant.hours;
+                document.getElementById('address').textContent = restaurant.address;
+                document.getElementById('phone').textContent = restaurant.phone;
+                document.getElementById('notes').textContent = restaurant.notes;
+            }
+            // to populate each section of the page with the proper data/ text  
+    </script>
+
+</body>
+</html>
```

Comment:
Similarly to the other HTML file, we want to contain all of this on our React side. The fetch looks like it would run correctly if we were to do this through HTML and JS but because we have this built out in react, this file isn't being used at all. Your restaurant page is being populated through your `Restaurant.js` file